modules/apps/object/object-test/src/testFunctional/tests/ObjectFields.testcase

```diff
@@ -2936,6 +2936,70 @@ definition {
 		}
 	}
 
+	@description = "LPS-164948 - Verify that the Formula Builder is not searchable"
+	@priority = 3
+	test FormulaFieldIsNotSearchable {
+		property custom.properties = "feature.flag.LPS-164948=true";
+
+		task ("Given: A Custom Object with two fields") {
+			ObjectAdmin.addObjectViaAPI(
+				labelName = "Custom Object 176845",
+				objectName = "CustomObject176845",
+				pluralLabelName = "Custom Objects 176845");
+
+			ObjectAdmin.openObjectAdmin();
+
+			ObjectPortlet.selectCustomObject(label = "Custom Object 176845");
+
+			ObjectAdmin.goToFieldsTab();
+
+			ObjectAdmin.addObjectFieldViaUI(
+				fieldLabel = "Custom Integer 1",
+				fieldType = "Integer");
```

Comment:
Why are we adding the field here via the UI when we're adding the same field via API in the next step?

Reply:
This doesn't make sense, I'll fix it.